src/Google.Api.Gax/ServiceSettingsBase.cs

```diff
@@ -44,6 +58,7 @@ public abstract class ServiceSettingsBase
         {
             settings.CallSettings = CallSettings.Clone();
             settings.Clock = Clock;
+            settings.UserAgent = UserAgent;
```

Comment:
I suspect this line isn't actually required, as the constructor will already have set it.
Could possibly replace this line with a comment saying why it's not necessary?